ui-src/src/components/Field.tsx

```diff
@@ -256,6 +256,10 @@ class Field extends React.Component<FieldProps, FieldState> {
           type: 'FETCH_ACTIONS',
           actions
         })
+        const playerHashes = actions.map((action: Action): Hash => {
+          return action.agentHash;
+        });
+        common.fetchIdentities(store.dispatch, playerHashes)
```

Comment:
Could move this to a lower-frequency polling loop

Reply:
...meaning we would have to store actions in state, or just as a member of this class